src/js/controller/captions.js

```diff
@@ -238,13 +242,7 @@ define([
                 });
             } else {
                 status = utils.tryCatch(function() {
```

Comment:
Look at the parser - I believe it throws when encountering error(s) - so you should try catch.

Also `utils.tryCatch` is a util that wraps execution in try catch blocks only in the release build. It's intended for event handling (noobs who write bad callbacks should not break the player). For code that handles valid exceptions use real `try {} catch(e) {}` blocks.